prime-dotnet-webapi/Helpers/SentryErrorReporter.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Threading.Tasks;
+
+using Prime;
+using SharpRaven;
+
```

Comment:
Reorder `using` statements?

Reply:
Actually `using Prime` isn't needed anymore after moving to namespace `Prime.Helpers`